wiki/quest-modding-modsbf.md

```diff
@@ -0,0 +1,21 @@
+# Quest modding using MBF
+:::danger Dev mode
+Developer mode needs to be enabled and the computer needs to be trusted.
+:::
+
+## What?
```

Comment:
Refer to how the Quest Patcher section is written for structuring instructions to getting started with this method